Can't install snarkjs

Hello, I can't install snarkjs

I'm running sudo npm install -g snarkjs to install it, with node version v14.17.1, and I get the following error:

```
npm ERR! code 128
npm ERR! command failed
npm ERR! command git --no-replace-objects ls-remote ssh://git@github.com/jbaylina/blake2b-wasm.git
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
```
I have an ssh key that is working fine, idk what I am missing, any help with this?

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
   },
   "dependencies": {
     "@iden3/binfileutils": "0.0.8",
-    "blake2b-wasm": "https://github.com/jbaylina/blake2b-wasm.git",
+    "blake2b-wasm": "^2.3.0",
     "circom_runtime": "0.1.13",
     "ejs": "^3.1.6",
     "fastfile": "0.0.19",

